src/server.py

```diff
@@ -287,6 +297,24 @@ def tracker_action_api(tracker_id: int) -> Response:
     return jsonify({})
 
 
+@app.route("/api/trackers/<int:tracker_id>/bots")
+def get_tracker_bots(tracker_id) -> Response:
+    status = request.args.get("status")
+    start = int(request.args.get("start", "0"))
```

Comment:
flask already has a built-in support for that:

```suggestion
    start = request.args.get("start", 0, type=int)
```

Reply:
I'll fix this in another PR, since there are 24 instances of this antipattern in this single file.